core/templates/dev/head/components/HintEditorDirective.js

```diff
@@ -65,7 +65,8 @@ oppia.directive('hintEditor', [
           };
 
           $scope.cancelThisHintEdit = function() {
-            $scope.hint = angular.copy($scope.hintMemento);
+            $scope.hint.hintContent =
```

Comment:
This looks good but I just want to check one more thing. Are there any other fields in hintMemento / hint that should be copied over?

Reply:
Only `hint.hintContent._html` property is modified on editing hint as can be seen [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/components/hint_editor_directive.html#L34). Using `$scope.hint.hintContent = angular.copy($scope.hintMemento.hintContent)` ensures that `_html` becomes same as previous value on clicking cancel while rest of the properties remain the same.

Reply:
OK, sounds good. Thanks!